src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ArrayInitHandler.java

```diff
@@ -91,7 +91,7 @@ protected DetailAST getListChild() {
     protected IndentLevel getChildrenExpectedIndent() {
         final IndentLevel expectedIndent =
             new IndentLevel(getIndent(), getIndentCheck().getArrayInitIndent(),
-                    getIndentCheck().getLineWrappingIndentation());
+                getLineWrappingIndentation());
```

Comment:
I assume this is just a code simplification change and not necessary to this issue?